app/resources/api/v2/plate_purpose_resource.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # Provides a JSON API representation of Plate
+    # See: http://jsonapi-resources.com/ for JSONAPI::Resource documentation
+    class PlatePurposeResource < BaseResource
+      model_name 'PlatePurpose'
+
+      # This resource is similar to PurposeResource but it was created to
+      # migrate the registration of plate purposes done by the Limber rake
+      # task config:generate from API version 1 to API version 2.
+
+      # The following attributes are sent by Limber for a new plate purpose.
+
+      # @!attribute name
+      #  @return [String] the name of the plate purpose
+      attribute :name
```

Comment:
It's probably a personal thing, but I don't find these kind of comments helpful - you're not really saying anything in lines 16 and 17 that isn't obvious from reading line 18 (same for most of the attributes below).

Reply:
I agree it is very chatty to describe a single attribute but I definitely want to see the type information and a short description. I can remove "@!attribute" lines so repeating names is avoided. It will not be fully compliant but it may serve the purpose. See: https://rubydoc.info/gems/yard/file/docs/Tags.md#attribute